README.md

```diff
@@ -25,3 +25,13 @@ For each features or contribution on the project :
 [feature request](https://github.com/ESGI-IW/Unsecure/issues/new?template=feature.md)
 
 ## Installation Unsecure
+
+* If on windows, install git bash for windows
```

Comment:
The README file is not completed, please checkt it @kosseylaHamouche

Reply:
👍